modules/algebrasAndModel/src/main/scala/models/Repository.scala

```diff
@@ -0,0 +1,6 @@
+package models
+import utils.Utils._
+
+sealed trait GitHubRepository
```

Comment:
```suggestion
sealed trait GitHubRepository extends Product with Serializable
```

Reference: https://typelevel.org/blog/2018/05/09/product-with-serializable.html

Reply:
I will have a look at the Reference, then make the necessary changes.